recipes/phidl/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "phidl" %}
+{% set version = "1.4.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/phidl-{{ version }}.tar.gz
+  sha256: b2b1b9bd6d92fac6fc1028a69d410e220e712785e13ef6286a162c2a8976881f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >3.6
+  run:
+    - gdspy >=1.5
+    - matplotlib-base
+    - numpy
+    - python
+    - six
+
+test:
+  imports:
+    - phidl
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/phidl/
+  doc_url: https://phidl.readthedocs.io/en/latest/tutorials.html
+  summary: PHIDL
+  license: MIT
+  license_file: LICENSE
```

Comment:
Add a comment that the license file is manually packaged. I've opened a PR upstream to deal with this in future releases (https://github.com/amccaugh/phidl/pull/86). The comment is a visual reminder that once this is merge, or some other solution is used, the manually packaged copy should be removed:

```
# License file manually packaged. See https://github.com/amccaugh/phidl/pull/86
license_file: LICENSE
...
```

Reply:
Thank you Joshua